.changes/unreleased/Under the Hood-20240327-001304.yaml

```diff
@@ -0,0 +1,6 @@
+kind: Under the Hood
+body: Lazy load agate
+time: 2024-03-27T00:13:04.246062-04:00
+custom:
+  Author: dwreeves
```

Comment:
@VersusFacit do you want to add your author?

Reply:
I'm on the merge commit :) but thank you